omod/src/main/java/org/openmrs/module/patientdashboard/web/controller/OPDEntryController.java

```diff
@@ -190,7 +190,7 @@ public String firstView(
        Person person=Context.getPersonService().getPerson(patientId);
 		Date birthday = patient.getBirthdate();
 		model.addAttribute("age", PatientUtils.estimateAge(birthday));
-		model.addAttribute("ageCategory", PatientUtils.estimateAgeCategory(birthday));
+		model.addAttribute("ageCategory", PatientUtils.estimateAge(birthday));

```

Comment:
This changes were reintroduced in hospital core to cover this use case https://github.com/uonafya/hospitalcoreopenmrs/blob/master/api/src/main/java/org/openmrs/module/hospitalcore/util/PatientUtils.java#L316 but I will accept that since it is overloaded to do the same thing